tests/test_risk_risiko.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from pytest_mock import MockerFixture
+from src.risk_risiko import risiko
+from src import risk_risiko
+
+
+def test_risiko(mocker: MockerFixture) -> None:
+    #act
+    ret : bool = risiko()
```

Comment:
quando il tipo è già specificato nella funzione non serve ridichiararlo, questione di "inference"
```suggestion
    ret = risiko()
```